biz.aQute.bndlib/src/aQute/bnd/build/model/BndEditModel.java

```diff
@@ -677,12 +685,72 @@ public void setIncludeSources(boolean includeSources) {
 	}
 
 	public List<String> getPrivatePackages() {
-		return doGetObject(aQute.bnd.osgi.Constants.PRIVATE_PACKAGE, listConverter);
+		List<String> privatePackagesEntries1 = getEntries(Constants.PRIVATEPACKAGE, listConverter);
+		List<String> privatePackagesEntries2 = getEntries(Constants.PRIVATE_PACKAGE, listConverter);
+
+		Set<String> privatePackages = Stream.concat(privatePackagesEntries1.stream(), privatePackagesEntries2.stream())
+			.collect(toSet());
```

Comment:
Since you make a stream, you don't need to collect into a set and then make a list from the set. Just use `distinct()` before the collect into list to remove duplicates.

Reply:
You are right. Fixed 👍